lib/new_relic/transaction/complete.ex

```diff
@@ -57,6 +58,20 @@ defmodule NewRelic.Transaction.Complete do
     })
   end
 
+  defp derive_queue_duration(%{start_time: start_time, queue_start_us: queue_start_us} = tx)
+       when not is_nil(queue_start_us) do
+    start_time_us = System.convert_time_unit(start_time, :native, :microsecond)
+    queue_duration_us = max(0, start_time_us - queue_start_us)
+
+    tx
+    |> Map.merge(%{
+      queue_duration_us: queue_duration_us,
+      queue_duration_s: queue_duration_us / 1_000_000
```

Comment:
This is actually where we need to report the final `queueDuration` attribute... I don't think we need the other time unit attributes, and we should actually drop the `queue_start_us` attribute

Reply:
Oh. So these transaction attributes will be added to the final transaction automatically? The key should be queueDuration in that case?